src/components/Landing.js

```diff
@@ -211,8 +215,25 @@ export default class Landing extends Component {
       }
     }
     alerts.sort(function (a, b) {
-      return b.priority - a.priority;
+      return a.priority - b.priority;
     });
+
+    //process graph data
+    if (overviewSection.graphConfig && overviewSection.graphConfig.summaryDataSource) {
+      //get the data from summary data
+      let sections = overviewSection.graphConfig.summaryDataSource;
+      let graph_data = [];
+     
+      sections.forEach(item => {
+        if (summary[item.section_key] && summary[item.section_key][item.subSection_key]) {
+          //console.log("section data? ", summary[item.section_key][item.subSection_key])
+          graph_data = [...graph_data, ...summary[item.section_key][item.subSection_key]];
```

Comment:
Cool, wasn't aware of [rest parameters](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Functions/rest_parameters)!